var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -80,32 +80,57 @@ def install(self, spec, prefix):
             optflags = {
                 'gcc': ['-O2',
                         '-ffast-math',
-                        '-ffree-form',
-                        '-ffree-line-length-none',
                         '-ftree-vectorize',
                         '-funroll-loops',
                         '-mtune=native'],
                 'intel': ['-O2',
                           '-pc64',
                           '-unroll',
-                          '-heap-arrays 64']
+                          '-xHost']
```

Comment:
If you want to be super conservative, when using the -ax option, you should also specify -xSSE2. This will set the default vectorization path to use SSE2. You could also select something greater than SSE2 for you -x option if you know you have a higher baseline. I am not a fan of the -xHost option. This messed up our Python installations here at LLNL since we have shared installations that run on multiple hosts and I was using the latest-greatest Intel to build things. I don't think that at the Spack level any package should have -xHost. Some build systems will default to this, but that is outside our purview.